arangod/IResearch/IResearchDataStore.cpp

```diff
@@ -869,6 +879,27 @@ Result IResearchDataStore::commitUnsafeImpl(
 
       commitLock.lock();
     }
+
+#ifdef USE_ENTERPRISE
+    bool const reopenColumnstore = [&] {
```

Comment:
Why do we need lamda here? It is called only once. I suggest to just calculate boolean here.

Reply:
To make it const + scope looks good for me